client/cli/src/params/database_params.rs

```diff
@@ -29,10 +29,16 @@ pub struct DatabaseParams {
 	/// Limit the memory the database cache can use.
 	#[arg(long = "db-cache", value_name = "MiB")]
 	pub database_cache_size: Option<usize>,
+
+	/// Required available space on database storage. If available space for DB storage drops below
+	/// the given threshold, node will be gracefully terminated. If `0` is given monitoring will be
+	/// disabled.
```

Comment:
Not sure if `0`-means-disabled convention is acceptable here. The other approach could be dedicated flag `--disable-storage-monitor`.
I am also open for better naming.